client/src/components/Tea/TeaEditor/TeaEditor.jsx

```diff
@@ -1,62 +1,46 @@
 /* eslint-disable no-console */
 import React from "react";
 import { Link } from "react-router-dom";
-import DataList from "../../DataList";
+import { InputField } from "../../FormComponents/InputField";
 
-export const TeaEditor = props => {
+export const TeaEditor = React.memo(props => {
```

Comment:
Unfortunately, this is still going to re-render each time, since the props it is passed change every time the input is changed.  You can see this by sticking a console.log or debugger statement in here.  If you want to avoid re-rendering this whole component every time TeaEditorContainer passes it new props, you'll need to make it a `React.Component` and implement your own `shouldComponentUpdate` method that ignores props that are only being passed to the `InputField`s and are not important for the `TeaEditor` parent component.